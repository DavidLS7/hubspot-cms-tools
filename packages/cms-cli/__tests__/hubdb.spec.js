/*global banjo */

describe('HubDB command', () => {
  it('works with help positional', async () =>
    expect(await banjo(['hubdb', 'help'])).toMatchSnapshot());

  it('works with help option', async () =>
    expect(await banjo(['hubdb', '--help'])).toMatchSnapshot());
});
