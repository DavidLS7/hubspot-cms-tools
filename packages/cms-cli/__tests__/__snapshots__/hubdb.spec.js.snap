// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HubDB command works with help option 1`] = `
"banjo hubdb

Manage HubDB tables

Commands:
  banjo hubdb clear <tableId>         clear all rows in a HubDB table
  banjo hubdb create <src>            Create a HubDB table
  banjo hubdb fetch <tableId> [dest]  fetch a HubDB table
  banjo hubdb delete <tableId>        delete a HubDB table

Options:
  --version     Show version number                                    [boolean]
  --debug, -d   set log level to debug                [boolean] [default: false]
  --help        Show help                                              [boolean]
  --config, -c  path to a config file                                   [string]
  --portal, -p  HubSpot portal id or name from config                   [string]
"
`;

exports[`HubDB command works with help positional 1`] = `
"banjo hubdb

Manage HubDB tables

Commands:
  banjo hubdb clear <tableId>         clear all rows in a HubDB table
  banjo hubdb create <src>            Create a HubDB table
  banjo hubdb fetch <tableId> [dest]  fetch a HubDB table
  banjo hubdb delete <tableId>        delete a HubDB table

Options:
  --version     Show version number                                    [boolean]
  --debug, -d   set log level to debug                [boolean] [default: false]
  --help        Show help                                              [boolean]
  --config, -c  path to a config file                                   [string]
  --portal, -p  HubSpot portal id or name from config                   [string]
"
`;
